src/sql/workbench/contrib/notebook/browser/notebookEditor.component.html

```diff
@@ -6,4 +6,5 @@
 -->
 <!-- Notebook View -->
 <notebook-component *ngIf="viewMode === 0" [_views]="views" [_model]="model"></notebook-component>
-<!-- Dashboard View: To be implemented-->
+<!-- Dashboard View -->
+<notebook-view-component *ngIf="viewMode === 1" [views]="views" [notebookMeta]="views.metadata" [activeView]="activeView" [model]="model"></notebook-view-component>
```

Comment:
Can you not use the actual ViewMode enum here? That would be much easier to understand. 

Also would suggest using the full name `notebookMetadata` here

Reply:
Do you mean 'ngIf="viewMode === ViewMode.Views"'? I had tried this before. It doesn't seem to work, and I couldn't find an example of one that did.

Renamed to notebookMetadata.

Reply:
Try using a local var : https://marco.dev/enums-angular

Reply:
That's a good idea. Didn't think of that. Done.